src-electron/importexport/import-json.js

```diff
@@ -57,10 +58,39 @@ function getPkgPath(pkg, zapFilePath) {
   }
 }
 
+// AutoLoad package. If succesful it returns an object:
+//  {
+//      packageId: <pkgId>
+//      packageType: <type>
+// }
```

Comment:
We may want to be consistent with our comments under
/* */

Reply:
Good point. I pushed another commit that cleans up this comment.